services/requester/requester.go

```diff
@@ -408,6 +411,15 @@ func (e *EVM) GetCode(
 	return code, nil
 }
 
+func (e *EVM) GetLatestBlockHeight(ctx context.Context) (uint64, error) {
+	blockHeader, err := e.client.GetLatestBlockHeader(ctx, true)
+	if err != nil {
+		return 0, err
+	}
+
+	return blockHeader.Height, nil
```

Comment:
```suggestion
	return e.client.GetLatestBlockHeader(ctx, true)
```

Reply:
The `GetLatestBlockHeader` returns a value of type `*flow.BlockHeader`, while we need to return the height included in the block header, which is of type `uint64`